actions/src/sema4ai/actions/agent/__init__.py

```
@@ -0,0 +1,286 @@
+
+    Returns:
+        The created conversation.
+    """
+    client = _AgentAPIClient(api_key=sema4_api_key)
+
+    endpoint = f"agents/{agent_id}/conversations"
+    full_url = urljoin(client.api_url, endpoint)
+    log.info(f"Agent Server API Call URL: {full_url}")
+
+    response = client.request(
+        endpoint,
+        method="POST",
+        json_data={"name": conversation_name},
+    )
+
+    response_json = response.json()
+    return Conversation.model_validate(response_json)
+
+
+def send_message(
+    conversation_id: str, agent_id: str, message: str, sema4_api_key: str | None = None
+) -> str:
+    """Sends a message within a conversation and retrieves the agent's response.
+
+    Args:
+        conversation_id: The ID of the conversation
+        agent_id: The ID of the agent to send message to
+        message: The message content to send
+        sema4_api_key: The API key for the Sema4 API if running in cloud. Use LOCAL if in Studio or SDK!
+
+    Returns:
+        Response containing either the agent's response or an error message
+    """
+    client = _AgentAPIClient(api_key=sema4_api_key)
+
+    # Handle case where conversation_id contains full path information
+    conversation_id_only = conversation_id
+    if "/" in conversation_id:
+        parts = conversation_id.split("/")
+        # Check if this is in the format agents/{agent_id}/conversations/{conversation_id}
+        if len(parts) >= 4 and "conversations" in parts:
+            conv_index = parts.index("conversations") + 1
+            if conv_index < len(parts):
+                conversation_id_only = parts[conv_index]
+                log.info(f"Extracted conversation_id: {conversation_id_only} from path")
+            else:
+                # Invalid path format
+                raise AgentApiClientException(
+                    f"Invalid path format in conversation ID: {conversation_id}"
+                )
```

Isn't there a case where this might fail? Maybe raise an exception?
`if len(parts) < 4 or ...`